amgut/db/patches/0036.sql

```diff
@@ -0,0 +1,389 @@
+-- May 31, 2016
+-- Add surfer and fermeted foods questionnaires
+
+----------------------------------------------------------
+-- survey_group
+----------------------------------------------------------
+INSERT INTO survey_group (group_order, american, british) VALUES (-3, 'Fermented Foods', 'Fermented Foods');
```

Comment:
To keep style and layout with all the other patches that add/modify survey information.